autozone-poc/src/utlis/slick_slider/SimpleSlider.js

```diff
@@ -0,0 +1,164 @@
+import React, { Component } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+export default class SimpleSlider extends Component {
```

Comment:
Why don't use functional component instead of class component

Reply:
Class component has been changed onto function component